pkg/epforwarder/epforwarder.go

```diff
@@ -215,7 +226,7 @@ func newHTTPPassthroughPipeline(desc passthroughPipelineDesc, destinationsContex
 	if endpoints.BatchMaxSize <= pkgconfig.DefaultBatchMaxSize {
 		endpoints.BatchMaxSize = desc.defaultBatchMaxSize
 	}
-	if endpoints.InputChanSize <= 0 {
+	if endpoints.InputChanSize <= pkgconfig.DefaultInputChanSize {
```

Comment:
Note: Need to use similar logic than `BatchMaxSize` and `BatchMaxContentSize` above. 
At this point `endpoints.InputChanSize` contain the default value set in config.go, comparing `<= 0` won't lead to expected result that is using `pkgconfig.DefaultInputChanSize` by default.